azafea/tests/test_cli.py

```diff
@@ -27,7 +27,7 @@ def process(*args, **kwargs):
 
         return process
 
-    config_file = make_config_file({'queues': {'some-queue': {'handler': 'azafea.tests.test_cli'}}})
+    config_file = make_config_file({'queues': {'some-queue': {'handler': 'azafea.tests'}}})
```

Comment:
I don't understand this change.

> **tests: Make test queue handlers a package**
>
> Azafea is setup to expect the handler methods (`process` and
> `register_commands`) in a package with `migrations` as a sub-package.
> Otherwise newer alembic blows up when it tries to look for the
> `migrations` sub-package when it's the handler is defined as a module.

In this commit you're just changing these config files but you're not making any changes to `azafea.tests`. And that namespace doesn't define those two functions or the sub-package `migrations`.  How can this work / do anything?

Reply:
I wrote that commit message hastily. What's happening is `get_callable` is being mocked just above that so that it magically returns a `process` method without having to add one to the actual package/module specified in `handler`. In essence, this is a fake handler and what's being specified doesn't matter except that it's an importable path. Most of the cases I think would be fine left as is, but I changed them all to be consistent since specifying a leaf module instead of a package would not be legitimate.

Where it actually matters is when alembic gets involved via `make-migration` or `migratedb`. In that case, we dynamically tell alembic that the migrations directories (as configured with `version_locations`) [are relative to the handler module](https://github.com/endlessm/azafea/blob/593f5fedb8d8d818534ef771296bf20a8202dfb0/azafea/migrations/utils.py#L26). If the "module" isn't a directory (i.e., a package), then alembic (or maybe our code driving alembic) fails.

Reply:
If I'm being honest, I didn't really understand the issue completely until I wrote that comment.